examples/HttpListenerExample/include/HttpListener.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef HTTPLISTENER_HPP
+#define HTTPLISTENER_HPP
+
+#include <IListener.hpp>
+
+namespace api::listeners
+{
+    class HttpListener : public IListener
+    {
+        public:
+            HttpListener();
```

Comment:
why not setup `Listener()` with callback as parameter and a default value ?

Reply:
When using this way we can change the callback at runtime